docgen/plugins/inlineProps/extractMetadata.js

```diff
@@ -28,6 +28,10 @@ const getCompiler = memoize(entriesArray => {
             join(__dirname, 'transformPropsLoader.js'),
           ],
         },
+        {
+          test: /\.css$/,
```

Comment:
Here we might just want to return an empty object every time we require some css. This should be sufficient for props/docs generation.